style.css

```diff
@@ -534,7 +534,6 @@ a {
 }
 
 @media screen and (min-width: 768px) {
-
   .header {
```

Comment:
Great design @Ramaunspoken 🤝

- [ ] Kindly make sure your design is a close replica of the provided design. You may align your nav bars as provided in the design.
- Your design
![your-desktop-nav](https://user-images.githubusercontent.com/63917725/204551626-9880b16c-e324-4da8-b6ee-9f1007209605.PNG)
- Provided design
![given-nav](https://user-images.githubusercontent.com/63917725/204551716-0afe4f6c-3655-452d-b572-b24749cb1e11.PNG)